public/css/nexthome.css

```diff
@@ -644,99 +570,70 @@ footer .container .right span.green {
     left: calc(5% + 7px);
     z-index: 99;
 }
-
 .img-size {
     height: 20px;
 }
-
 .networkStatsWrap,
 .blockRewardsWrapper,
 .proofofstakeWrapper {
     padding-right: 50px;
 }
-
 .info-block .expanded-block .row .col-12 img {
     margin-right: 3%;
     margin-top: 2%;
     height: 20px;
 }
-
 @media (min-width: 1400px) {
     .container {
         max-width: 1280px !important;
-    }
-    .container .logo a img {
+    }    .container .logo a img {
         width: 100px !important;
-    }
-}
-
+    }}
 @media (min-width: 800px) {
     header .right .searchbox.full input {
         flex: 1;
-    }
-}
-
+    }}
 @media (max-width: 600px) {
     header .right .searchbox input {
         width: 150px;
-    }
-}
-
+    }}
 @media only screen and (max-width: 768px) {
     .container {
         max-width: 85% !important;
-    }
-    .container .logo a img {
+    }    .container .logo a img {
         width: 70px !important;
-    }
-    .col-md-6 {
+    }    .col-md-6 {
         flex: 0 0 50%;
         max-width: 100%;
-    }
-    .col-sm-4 {
+    }    .col-sm-4 {
```

Comment:
same all over the CSS files.  I get removing the numerous empty lines, but starting a new style on the same line like this makes it far less legible.